src/Components/Atoms/FATooltip/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { memo } from "react";
+import { Tooltip } from "@mui/material";
+import { IFATooltip } from "./FATooltip.interface";
+
+export const FATooltip = memo((props: IFATooltip) => {
+  const { testID, title, ...materialUIProps } = props;
```

Comment:
`title` is a dynamic value, so remove it from this component.

Reply:
done beautifully